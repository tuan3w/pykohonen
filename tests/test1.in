10 10 0.1 2 10
10 10 0.1 4 10
10 10 0.1 6 10
10 10 0.5 2 10
10 10 0.5 4 10
10 10 0.5 6 10
10 10 0.9 2 10
10 10 0.9 4 10
10 10 0.9 6 10
15 15 0.1 2 10
15 15 0.1 4 10
15 15 0.1 6 10
15 15 0.5 2 10
15 15 0.5 4 10
15 15 0.5 6 10
15 15 0.9 2 10
15 15 0.9 4 10
15 15 0.9 6 10
20 20 0.1 2 10
20 20 0.1 4 10
20 20 0.1 6 10
20 20 0.5 2 10
20 20 0.5 4 10
20 20 0.5 6 10
20 20 0.9 2 10
20 20 0.9 4 10
20 20 0.9 6 10
0 0 0 0 0
