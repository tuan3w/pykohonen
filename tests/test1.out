O acerto para confinado foi: 97.5609756098
O acerto para nao confinado foi: 5.71428571429
O acerto total da RNA foi: 87.5776397516



O acerto para confinado foi: 58.5365853659
O acerto para nao confinado foi: 28.5714285714
O acerto total da RNA foi: 55.2795031056



O acerto para confinado foi: 100.0
O acerto para nao confinado foi: 0.0
O acerto total da RNA foi: 89.1304347826



O acerto para confinado foi: 99.3031358885
O acerto para nao confinado foi: 2.85714285714
O acerto total da RNA foi: 88.8198757764



O acerto para confinado foi: 100.0
O acerto para nao confinado foi: 0.0
O acerto total da RNA foi: 89.1304347826



O acerto para confinado foi: 59.2334494774
O acerto para nao confinado foi: 20.0
O acerto total da RNA foi: 54.9689440994



O acerto para confinado foi: 65.5052264808
O acerto para nao confinado foi: 34.2857142857
O acerto total da RNA foi: 62.1118012422



O acerto para confinado foi: 41.1149825784
O acerto para nao confinado foi: 34.2857142857
O acerto total da RNA foi: 40.3726708075



O acerto para confinado foi: 23.6933797909
O acerto para nao confinado foi: 91.4285714286
O acerto total da RNA foi: 31.0559006211



O acerto para confinado foi: 82.2299651568
O acerto para nao confinado foi: 28.5714285714
O acerto total da RNA foi: 76.397515528



O acerto para confinado foi: 91.9860627178
O acerto para nao confinado foi: 14.2857142857
O acerto total da RNA foi: 83.5403726708



O acerto para confinado foi: 58.1881533101
O acerto para nao confinado foi: 40.0
O acerto total da RNA foi: 56.2111801242



O acerto para confinado foi: 85.0174216028
O acerto para nao confinado foi: 17.1428571429
O acerto total da RNA foi: 77.6397515528



O acerto para confinado foi: 78.0487804878
O acerto para nao confinado foi: 28.5714285714
O acerto total da RNA foi: 72.6708074534



O acerto para confinado foi: 72.1254355401
O acerto para nao confinado foi: 25.7142857143
O acerto total da RNA foi: 67.0807453416



O acerto para confinado foi: 98.2578397213
O acerto para nao confinado foi: 5.71428571429
O acerto total da RNA foi: 88.198757764



O acerto para confinado foi: 99.6515679443
O acerto para nao confinado foi: 2.85714285714
O acerto total da RNA foi: 89.1304347826



O acerto para confinado foi: 85.3658536585
O acerto para nao confinado foi: 31.4285714286
O acerto total da RNA foi: 79.5031055901



O acerto para confinado foi: 90.9407665505
O acerto para nao confinado foi: 17.1428571429
O acerto total da RNA foi: 82.9192546584



O acerto para confinado foi: 70.7317073171
O acerto para nao confinado foi: 48.5714285714
O acerto total da RNA foi: 68.3229813665



O acerto para confinado foi: 87.8048780488
O acerto para nao confinado foi: 17.1428571429
O acerto total da RNA foi: 80.1242236025



O acerto para confinado foi: 76.3066202091
O acerto para nao confinado foi: 37.1428571429
O acerto total da RNA foi: 72.049689441



O acerto para confinado foi: 67.2473867596
O acerto para nao confinado foi: 54.2857142857
O acerto total da RNA foi: 65.8385093168



O acerto para confinado foi: 99.6515679443
O acerto para nao confinado foi: 2.85714285714
O acerto total da RNA foi: 89.1304347826



O acerto para confinado foi: 86.7595818815
O acerto para nao confinado foi: 22.8571428571
O acerto total da RNA foi: 79.8136645963



O acerto para confinado foi: 66.8989547038
O acerto para nao confinado foi: 54.2857142857
O acerto total da RNA foi: 65.5279503106



O acerto para confinado foi: 99.6515679443
O acerto para nao confinado foi: 2.85714285714
O acerto total da RNA foi: 89.1304347826



